classes/Search.php

```diff
@@ -349,7 +349,7 @@ public static function find(
         $sqlScore = '';
         if (!empty($scoreArray) && is_array($scoreArray)) {
             $sqlScore = ',( ' .
-                'SELECT SUM(weight) ' .
```

Comment:
could you please explain this fix ? here you'll get a negative value instead of a positive one, right ? if yes, why is that ?